data/json/items/armor.json

```diff
@@ -706,64 +706,58 @@
     "flags": [ "BELTED", "WATER_FRIENDLY" ]
   },
   {
-    "id": "badge_cybercop",
+    "abstract": "badge_abstract",
     "type": "ARMOR",
-    "name": "cybercop shield",
-    "description": "A chromed silver shield identifies the wearer as an authority even the cop-killers couldn't kill.",
+    "name": "badge template",
+    "description": "This is a template for police badges.  If found in a game it is a bug.",
     "weight": 60,
     "volume": 0,
     "price": 5000,
     "material": [ "silver" ],
     "symbol": "[",
-    "color": "yellow"
+    "color": "yellow",
+    "covers": [ "TORSO" ],
+    "coverage": 2,
+    "material_thickness": 2,
+    "flags": [ "ALLOWS_NATURAL_ATTACKS", "OUTER", "OVERSIZE", "WATER_FRIENDLY" ]
```

Comment:
Actually, I just realized; removing outer will result in no penalty when worn with other outer equipment, but will result in a penalty when worn with the normal layer equipment (for example a t-shirt)… This might be the case why those decoration items didn't cover a specific body part originally, else they give encumbrance when layered with other items in the current system…

I guess there's no tag to prevent the encumbrance penalty (yet) (or is there any?), so the question is, should I remove the `covers` parameter altogether? If so, what will the coverage with cover assigned do? Protect all body parts? Currently ear plugs have coverage of 1% but cover no particular part of clothing and I don't know how it works (asked that on Discord earlier today, but got no reply).

Reply:
Yeah, stuff like badges and jewelry and such that aren't supposed to cause actual encumbrance aren't assigned to a body part for that exact reason.